### PR TITLE
Bipen doing abilities for all classes

### PR DESCRIPTION
**Pull request by BipenTheBat**

### CODE REVIEW

File: assets/data/aspects/drauvenSkillDeck.json [hunk 1]
```diff
@@ -1,37 +1,164 @@
 [
-  {
+{
+	"modId": "wildermyth-drauven-pcs",
+	"id": "drauvenDeck_battleyell",
+	"effects": [ "drauven_pc_battleyell" ],
+	"boostType": "ABILITY_CLASS",
+	"legacyAbilityChoice": true,
+	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
+	"info": { "abilityDeckUsage": "common", "abilityDeckIcon": "bloodrage" }
```
Comment (BoldestDungeon) [inline assets/data/aspects/drauvenSkillDeck.json:9]: Is this common or warrior?

File: assets/data/aspects/drauvenSkillDeck.json [hunk 2]
```diff
@@ -1,37 +1,164 @@
...
+},
+{
+	"modId": "wildermyth-drauven-pcs",
+	"id": "drauvenDeck_battleyell_upgrade",
+	"legacyAbilityChoice": true,
+	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
+	"info": {
+		"abilityDeckUsage": "upgrade",
+		"upgradeForAbility": "drauvenDeck_battleyell",
+		"abilityDeckIcon": "ignite"
```
Comment (BoldestDungeon) [inline assets/data/aspects/drauvenSkillDeck.json:19]: Should this be the same `abilityDeckIcon` as the base ability (`"bloodrage"`)?

File: assets/data/aspects/drauvenSkillDeck.json [hunk 3]
```diff
@@ -1,37 +1,164 @@
...
+	}
+},
+{
+	"modId": "wildermyth-drauven-pcs",
+	"id": "drauvenDeck_cower",
+	"effects": [ "drauven_pc_cower" ],
+	"boostType": "ABILITY_CLASS",
+	"legacyAbilityChoice": true,
+	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
+	"info": { "abilityDeckUsage": "hunter", "abilityDeckIcon": "ignite" }
```
Comment (BoldestDungeon) [inline assets/data/aspects/drauvenSkillDeck.json:79]: There might be a better icon for this one - maybe the rogue one?

File: assets/data/aspects/drauvenSkillDeck.json [hunk 4]
```diff
@@ -1,37 +1,164 @@
...
+	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
+	"info": {
+		"abilityDeckUsage": "upgrade",
+		"upgradeForAbility": "drauvenDeck_specterstep",
+		"abilityDeckIcon": "silkstepCloak"
+	}
+},
+{
+	"modId": "wildermyth-drauven-pcs",
+	"id": "drauvenDeck_upgraded_bloodedFerocity",
```
Comment (BoldestDungeon) [inline assets/data/aspects/drauvenSkillDeck.json:154]: I've been sticking to a pattern of `drauvenDeck_baseSkill` and `drauvenDeck_baseSkill_upgrade` to try and keep things consistent - if it's not too nitpicky, let's maybe keep the names consistent (and maybe move this to live beside the base ability definition?)

File: assets/data/effects/drauven/drauven_pc_preparedShot.json [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+		"missionFeedback": { "suggestionFeedback": "FRIENDLY_EFFECT_RANGE" },
+		"aiHints": {
+			"scoreNearbyFoe": "100",
+			"foeRange": "2",
+			"foeMaxRange": "10",
+			"scoreNearbyFriend": "10",
+			"friendRange": "2"
+		},
+		"minRange": "0.9",
+		"range": "(5.6-1)+self.BONUS_RANGE",
```
Comment (BoldestDungeon) [inline assets/data/effects/drauven/drauven_pc_preparedShot.json:33]: Isn't this just equivalent to `4.6 + self.BONUS_RANGE`? What does the `-1` represent?

File: assets/data/effects/drauven/drauven_pc_quickblade.json [hunk 6]
```diff
@@ -0,0 +1,52 @@
+{
+"id": "drauven_pc_quickblade",
+"info": {
+	"dataVersion": 1,
+	"sourceFile": "drauven/drauven_pc_quickblade",
+	"modId": "wildermyth-drauven-pcs",
+	"author": "timcrosby",
```
Comment (BoldestDungeon) [inline assets/data/effects/drauven/drauven_pc_quickblade.json:7]: Can update the author in this file :)

File: assets/data/effects/drauven/drauven_pc_upgraded_bloodedFerocity.json [hunk 7]
```diff
@@ -0,0 +1,48 @@
+{
```
Comment (BoldestDungeon) [inline assets/data/effects/drauven/drauven_pc_upgraded_bloodedFerocity.json:1]: To keep this and the parent files adjacent in the folder view, I would recommend naming it `drauven_pc_bloodedFerocity_upgraded`

**Review by BoldestDungeon — APPROVED** [review]
Overall looks good! A few minor nitpick comments for you, if you want :)

File: assets/data/effects/drauven/drauven_pc_preparedShot.json (hunk 5 above)
Comment (BipenTheBat) [reply assets/data/effects/drauven/drauven_pc_preparedShot.json:33]: need that -1 or wont work

**Pull request merged**